waspc/src/Wasp/AppSpec/Entity.hs

```diff
@@ -54,18 +50,5 @@ getPslModelBody = pslModelBody
 getIdField :: Entity -> Maybe Psl.Model.Field
 getIdField = findIdField . getPslModelBody
 
-isFieldUnique :: String -> Entity -> Maybe Bool
-isFieldUnique fieldName = doesFieldHaveAttribute fieldName "unique"
-
-doesFieldHaveAttribute :: String -> String -> Entity -> Maybe Bool
```

Comment:
This was a weirdly specific function which I tried to rewrite but in the end it was easier just to use `doesPslFieldHaveAttribute` directly in the one place where we needed to check if a field has an attribute.